tests/plugins/clipboard.test.ts

```diff
@@ -88,6 +89,24 @@ describe("clipboard", () => {
     expect(getCell(model, "D3")).toBeUndefined();
   });
 
+  test("can clean the clipboard visible zones", () => {
+    const model = new Model();
+    setCellContent(model, "B2", "b2");
+    expect(getCell(model, "B2")).toMatchObject({
+      content: "b2",
+    });
```

Comment:
not useful for this test. We really don't care about the content here, just the outlined zone